packages/google_sign_in/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 4.0.5+1
```

Comment:
version should be 4.0.6
https://dart.dev/tools/pub/versioning#semantic-versions

Reply:
I was in doubt because of this:

> going to 0.1.2+1 indicates a change that doesn’t affect the public API.

Since my modification doesn't change the public API, shouldn't it be a `+1`?

Reply:
that is true for pre 1.0.0, after 1.0.0, in a x.y.z version, x is the major, y is the minor and z is the patch.
Before 1.0.0, it becomes 0.x.y+z

Reply:
Ohh... Then I think the `+` should be better explained in that documentation, as I see a lot of post-1.0.0 packages with `+`, like [`provider 3.0.0+1`](https://pub.dev/packages/provider). 

What's the meaning of `+1` after 1.0.0 then?

Reply:
There shouldn't be a +1 after 1.0.0. provider 3.0.0+1 is probably a mistake.